protocol/x/clob/keeper/clob_pair.go

```diff
@@ -468,13 +468,19 @@ func (k Keeper) UpdateClobPair(
 
 	oldStatus := oldClobPair.Status
 	newStatus := clobPair.Status
-	if oldStatus != newStatus && !types.IsSupportedClobPairStatusTransition(oldStatus, newStatus) {
-		return errorsmod.Wrapf(
-			types.ErrInvalidClobPairStatusTransition,
-			"Cannot transition from status %+v to status %+v",
-			oldStatus,
-			newStatus,
-		)
+	if oldStatus != newStatus {
```

Comment:
nit: move `oldStatus == newStatus` check to `types.IsSupportedClobPairStatusTransition`.  

transitioning to the same status is still a transition. if in the future we want to disallow transition to the state status for `INITIALIZING` or `FINAL_SETTLEMENT`, they should all be defined inside this transition function